docs/source/user-guide/configuration/free-channel.rst

```diff
@@ -0,0 +1,105 @@
+======================
+Using the free channel
+======================
+
+.. contents::
+   :local:
+   :depth: 2
+
+The free channel contains packages created prior to
+September 26, 2017. Prior to conda 4.7, the free
+channel was part of the ``defaults`` channel.
+Read more about the :ref:`defaults channel <default-channels>`.
+
+Removing the ``free`` channel reduced conda's search space
+and hid old software. That old software could have incompatible
+constraint information.
+
+If you still need the content from the ``free`` channel to reproduce
+old environments, you can re-add the channel following the directions below.
+
+.. _free-channel-default:
+
+Adding the free channel to defaults
+===================================
+
+If you want to add the ``free`` channel back into your default list,
+use the command::
+
+   conda config --set restore_free_channel true
+
+The order of the channels is important. Using the above
+command will restore the ``free`` channel in the correct order.
+
+Changing .condarc
+=================
+
+You can also add the ``free`` channel back into your defaults by
+changing the ``.condarc`` file itself.
+
+Add the following to the conda section of your ``.condarc`` file::
+   
+   restore_free_channel: true
```

Comment:
Does this always work? even if someone adds other channels like conda-forge to their defaults list?

Reply:
@msarahan  - Can you help with Crystal's question above please?

Reply:
Yes, it's a separate collection of stuff.  If people have set custom default_channels, it might not do the right thing.  I will check that.

Reply:
sorry, I haven't gotten to this yet.  I'll try to look at it today.